src/CrossScopeValidator.ts

```diff
@@ -618,16 +613,14 @@ export class CrossScopeValidator {
         }
 
         // Check each file for missing symbols - if they are missing in SOME scopes, add diagnostic
-        for (let [file, missingSymbolPerFile] of missingSymbolInScope.entries()) {
+        for (let [_, missingSymbolPerFile] of missingSymbolInScope.entries()) {
```

Comment:
Could we use `.values()` here now instead, since we aren't using the `file` item anymore? Should `missingSymbolInScope` structure be changed if we're not using file anymore? (I don't have enough context on what's being done here)